intellij-plugin-verifier/verifier-cli/src/main/java/com/jetbrains/pluginverifier/tasks/checkPlugin/CheckPluginResultPrinter.kt

```diff
@@ -24,11 +27,28 @@ class CheckPluginResultPrinter(private val pluginRepository: PluginRepository) :
         val teamCityHistory = printTcLog(true, outputOptions.teamCityLog, outputOptions.teamCityGroupType)
         outputOptions.postProcessTeamCityTests(teamCityHistory)
       } else {
-        printOnStdout(this)
+        if (outputOptions.usePlainOutput()) {
+          printOnStdout (this)
+        }
+      }
+
+      if (hasOnlyPluginsWithInvalidFiles()) {
```

Comment:
Comment on why MarkdownResultPrinter without a markdown option is used here may be useful.

Reply:
Thanks for spotting this. Markdown should be used only when explicitly requested. Fixed.